### ISSUE TITLE
Anyone had any success Data Sniffing UK?

### ISSUE THREAD

**Issue by aliguy** [sushant10/HQ_Bot#94]
It seems HQ have installed some new blocks on allowing devices to connect via a hotspot (computer etc.) - Has anyone has success in sniffing user id and bearer code from their iphone, specifically in UK?
Regards

**Comment by garethgtt** [comment]
i had a go but so far no- same with cash show cant find that either when i run the apps through a proxy

**Comment by aliguy** [comment]
Same here, Ill let you know if I find out how if you do ;)

**Comment by aliguy** [comment]
Any success my friend?

**Comment by garethgtt** [comment]
I only breifly tried- saw my phone posting auth details but didnt see any question info, - ive been concentrating on the question answering logic, did you have any luck?

**Comment by TJM4** [comment]
Nope, I've just been getting SSL errors when I try using Charles proxy despite having the certificates setup

**Comment by aliguy** [comment]
I’ve tried a combination of both local iPhone Https sniffers & hotspot ones
and still no luck - just sticking to reading the screen with other apps
like cash show atm
